flake8_implicit_str_concat.py

```diff
@@ -7,11 +7,24 @@
 """
 
 import ast
+import sys
 import tokenize
+from dataclasses import dataclass
 from typing import Iterable, List, Tuple
 
-import attr
-import more_itertools
+if sys.version_info >= (3, 10):
+    from itertools import pairwise
+else:
+    from itertools import tee
+    from typing import Iterator, TypeVar
+
+    _T = TypeVar("_T")
+
+    def pairwise(iterable: Iterable[_T]) -> Iterator[Tuple[_T, _T]]:
+        a, b = tee(iterable)
+        next(b, None)
+        yield from zip(a, b)
```

Comment:
Rather than maintaining our own copy of `pairwise` for Python 3.7-3.9, let's keep `more_itertools` for those and use the new `itertools` for Python 3.10, for example:

```suggestion
try:
    from itertools import pairwise
except ImportError:
    from more_itertools import pairwise
```

Reply:
try except ImporError does not work for mypy